rpcs3/Emu/RSX/gcm_enums.cpp

```diff
@@ -563,6 +563,7 @@ enum
 	CELL_GCM_TEXTURE_MAX_ANISO_16 = 7,
 
 	// Texture Filter
+	CELL_GCM_TEXTURE_FILTER_NONE = 0,
```

Comment:
Whatever they do, CELL_GCM_TEXTURE_FILTER_NONE simply doesn't exist. We didn't invent these constants, it's confusing. And note my recent issue about gcm_enums, this problem seems relevant that the values are checked in the situation they shouldn't.